staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -99,19 +99,35 @@ func unauthorized(w http.ResponseWriter, req *http.Request) {
 // authorized_user_requests metric, and to avoid pushing actual usernames in the
 // metric.
 func compressUsername(username string) string {
-	switch {
-	// Known internal identities.
-	case username == "admin" ||
-		username == "client" ||
-		username == "kube_proxy" ||
-		username == "kubelet" ||
-		username == "system:serviceaccount:kube-system:default":
+	switch username {
+	// Known internal identities
+	case "admin",
```

Comment:
Which of these are deployment specific, vs. standard? Do we just need to make this list a configurable parameter rather than hardcoding?